docs/collecting-data/code-generation/index.md

```diff
@@ -8,14 +8,10 @@ sidebar_label: "Code Generation"
 This documentation only applies to [Snowplow BDP](https://snowplow.io/snowplow-bdp/). See the [feature comparison](/docs/feature-comparison/index.md) page for more information about the different Snowplow offerings.
```

Comment:
Since you are editing this file, I’ve noticed that this banner is not correctly implemented. Could you fix it? The `:::info` needs to be removed entirely, and instead the following needs to be added before `---`:

```
sidebar_custom_props:
  offerings:
    - bdp
 ```

cc @igneel64 — please see https://github.com/snowplow/documentation?tab=readme-ov-file#offerings.